hubspot.com.hubspot-txt.json

```diff
@@ -0,0 +1,20 @@
+{
+  "providerId": "hubspot.com",
+  "providerName": "HubSpot",
+  "serviceId": "hubspot-txt",
+  "serviceName": "HubSpot Site",
+  "version": 1,
+  "logoUrl": "https://domainconnect.org/wp-content/uploads/2017/02/HubSpot-768x223.png",
+  "description": "Enables a domain to work with HubSpot",
+  "variableDescription": "txtValue - txt record data value, txtHostValue - txt host value",
+  "hostRequired": false,
+  "records": [
+    {
+      "groupId": "txt",
+      "type": "TXT",
+      "host": "%txtHostValue%",
```

Comment:
The same consideration as above - this template could however use `@` as host will determine the subdomain in fact.